akka-docs/src/main/paradox/security/2017-02-10-java-serialization.md

```diff
@@ -27,7 +27,7 @@ Please subscribe to the [akka-security](https://groups.google.com/forum/#!forum/
 
 ### Severity
 
-The [CVSS](https://en.wikipedia.org/wiki/CVSS) score of this vulnerability is 6.8 (Medium), based on vector [AV:A/AC:M/Au:N/C:C/I:C/A:C/E:F/RL:TF/RC:C](https://nvd.nist.gov/cvss.cfm?calculator&version=2&vector=\(AV:A/AC:M/Au:N/C:C/I:C/A:C/E:F/RL:TF/RC:C\)).
+The [CVSS](https://en.wikipedia.org/wiki/CVSS) score of this vulnerability is 6.8 (Medium), based on vector [AV:A/AC:M/Au:N/C:C/I:C/A:C/E:F/RL:TF/RC:C](https://nvd.nist.gov/cvss.cfm?calculator&amp;version=2&amp;vector=\(AV:A/AC:M/Au:N/C:C/I:C/A:C/E:F/RL:TF/RC:C\)).
```

Comment:
Is this really correct? Existing link works both as [link in rendered html](https://doc.akka.io/docs/akka/current/security/2017-02-10-java-serialization.html) and when pasting into browser. The changed link doesn't work when pasting into browser. Do you mean that Paradox is replacing those `&amp;` with `&` when generating html?

Reply:
Paradox is not replacing `&amp;` with `&`, but when encoding the link as the value of the `href` attribute it seems the [neat thing is to encode it](https://stackoverflow.com/questions/3705591/do-i-encode-ampersands-in-a-href) (even though modern browsers will indeed understand both).

I guess you could argue encoding those characters should be the responsibility of paradox, and not happen in the source document, though.

Reply:
The background-backgroud to this change is that I had written some Alpakka code that parsed these documents as XML, and the XML parser gets confused by them. But that was a one-off experiment anyway and I'm happy to drop this change if it's controversial ;)

Reply:
ok